### PR TITLE
UI changes to adopt sc changes

### PR DESCRIPTION
**Pull request by shivamgarglab49**

### CODE REVIEW

File: src/dex-ui/services/HederaService/GovernorService.ts [hunk 1]
```diff
@@ -147,15 +147,15 @@ const sendCreateTransferTokenProposalTransaction = async (
...
+    .addString(linkToDiscussion)
     .addAddress(transferFromAddress)
     .addAddress(transferToAddress)
     .addAddress(tokenToTransferAddress)
-    .addInt256(amountToTransfer)
-    /** This is 'title' on the contract function */
-    .addString(description)
-    .addString(linkToDiscussion);
+    .addInt256(amountToTransfer);
+  return Promise.reject();
```
Comment (roshan-bisht-lab49) [inline src/dex-ui/services/HederaService/GovernorService.ts:157]: Looks like early return code below will not work or this is done for testing purpose.?

**Review by roshan-bisht-lab49 — CHANGES_REQUESTED** [review]

File: src/dex-ui/services/HederaService/GovernorService.ts (hunk 1 above)
Comment (shivamgarglab49) [reply src/dex-ui/services/HederaService/GovernorService.ts:157]: Sorry, my bad!

File: src/dex-ui/pages/CreateProposal/ProposalForms/TextProposalForm.tsx [hunk 2]
```diff
@@ -23,9 +23,11 @@ function TextProposalForm() {
   const handleCancelClick = () => navigate("/governance");
 
   async function onSubmit(data: TextProposalFormData) {
-    // TODO: Proposal Contract functions do not handle both title and description yet.
-    // description: data.description,
-    await governance.createProposal(ProposalType.Text, { title: data.title });
+    await governance.createProposal(ProposalType.Text, {
+      title: data.title,
+      description: data.description,
+      linkToDiscussion: "",
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/CreateProposal/ProposalForms/TextProposalForm.tsx:29]: We'll need to add the discussion link input component to the Text Proposal form in a follow up PR.

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Review by roshan-bisht-lab49 — APPROVED** [review]

File: src/dex-ui/pages/CreateProposal/ProposalForms/TextProposalForm.tsx (hunk 2 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/CreateProposal/ProposalForms/TextProposalForm.tsx:29]: @damiano-melcarne-lab49  I have addressed this under https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/pull/88, one this PR is merged I can merge mine.

cc: @shivamgarglab49

**Pull request merged**